src/main/java/step3/Car.java

```diff
@@ -0,0 +1,14 @@
+package step3;
+
+public class Car {
+
+    private int count;
+
+    public void tryMove(int count){
+        this.count += 4 <= count? 1 : 0;
```

Comment:
```suggestion
        if (4 <= count) {
            this.count++;
        }
```
언어에 따라서는 이렇게 간결한 형태를 선호하기도 하지만 한줄에 여러 연산자를 쓰는건 지양합니다.